modal/runner.py

```diff
@@ -455,7 +455,7 @@ class DeployResult:
     app_id: str
     app_page_url: str
     app_logs_url: str
-    warnings: list[str]
+    warnings: list[api_pb2.Warning]
```

Comment:
Meh since the `DeployResult` is public API we should probably coerce the proto message into a dataclass or some other representation that users are meant to interact with.